src/components/Captions/imagecaptions.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "styled-components";
+
+const CaptionContainer = styled.div`
+  display: flex;
+  flex-flow: wrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: 480px;
+  text-align: left;
+`;
+
+const CaptionTitle = styled.h4`
+  font-family: "Raleway";
+  font-style: normal;
+  font-weight: 800;
+  font-size: 30px;
+  line-height: 35px;
+  text-align: inherit;
+  color: #ffffff;
```

Comment:
Use a CSS variable for all colors

Reply:
@erinkelley27 just waiting for them to be made but will add them in once I can